### PR TITLE
CITATION tech note

### PR DESCRIPTION
**Pull request by maelle**
Not sure whether/how to also mention https://github.com/Pakillo/grateful and https://github.com/dirkschumacher/thankr that are more for users, not developers.

### CODE REVIEW

File: content/technotes/2021-03-02-package-citation/index.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+<!--html_preserve-->
+{{< figure src ="pexels-cottonbro-4553166.jpg" alt = "Person wearing a cardboard box on their head" link = "https://www.pexels.com/photo/person-in-white-long-sleeve-shirt-holding-brown-cardboard-box-4553166/" caption = "Why didn't you cite my package?! [cottonbro on Pexels](https://www.pexels.com/photo/person-in-white-long-sleeve-shirt-holding-brown-cardboard-box-4553166/)." class = "center" >}}
+<!--/html_preserve-->
+
+To encourage users to cite your package, you need to store citation metadata in the expected places and to advertise it very clearly.
+
+> "I [advertise the citation info in the README] too, just to make it painfully obvious how to cite the work." [Adam Sparks](/author/adam-sparks/), [rOpenSci forum](https://discuss.ropensci.org/t/should-the-ropensci-dev-guide-include-package-citation-best-practices/1551/18)
+
+* Use the CITATION file, as [thousands of R packages do](https://github.com/search?q=user%3Acran+path%3A%2Finst%2F+filename%3ACITATION&type=Code&ref=advsearch&l=&l=). Create its boilerplate with `usethis::use_citation()` for instance.
```
Comment (mpadge) [inline content/technotes/2021-03-02-package-citation/index.md:36]: 3,386 from from 16,002 as of 15 Jan 2021

Comment (maelle) [reply content/technotes/2021-03-02-package-citation/index.md:36]: thank you Mark!!

File: content/technotes/2021-03-02-package-citation/index.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+description: Some tips to promote scientific citations of your work.
+tags:
+  - citations
+  - metadata
+  - packages
+---
+
+> Software citation elevates software to the level of a first-class object in the digital scholarly ecosystem, consistent with its immense actual present-day significance. Katz et al., F1000Research 2021. [^f1000]
+
+As a package author you are probably happy to hear about its usage, and within academia, even more so if it is done with the usual currency of citations in scientific articles.
```
Comment (sckott) [inline content/technotes/2021-03-02-package-citation/index.md:18]: Is "its" in "about its usage" unclear as to what "it" refers to?

File: content/technotes/2021-03-02-package-citation/index.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+{{< tweet 1273605411577442304 >}}
+
+In this post, we shall share tips on how to make your R package easy to cite.
+We shall also give some insights on obstacles the users of your package might face, as well as a brief presentation a our own literature surveillance.
```
Comment (sckott) [inline content/technotes/2021-03-02-package-citation/index.md:23]: "brief presentation on our own"

File: content/technotes/2021-03-02-package-citation/index.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+  " nasapower: A NASA POWER Global Meteorology, Surface Solar Energy and Climatology Data Client for R. ",
+  note, ".",
+  " https://CRAN.R-project.org/package=nasapower")
+```
+
+* Direct potential readers to the preferred citation in the README by adding boilerplate text "here's how to cite my package". See e.g. [ecmwfr README](https://github.com/bluegreen-labs/ecmwfr#how-to-cite-this-package-in-your-article).
+
+{{< tweet 1261726909148233728 >}}
+
+Something we at rOpenSci wouldn't recommend is an on load message in interfactive session, but it's good to know that [some maintainers resort to this strategy](https://twitter.com/d_olivaw/status/1261439521134182406).
```
Comment (sckott) [inline content/technotes/2021-03-02-package-citation/index.md:91]: interactive

File: content/technotes/2021-03-02-package-citation/index.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+* Co-authors less willing to cite software than they are.
+
+{{< tweet 1197124137078136832 >}}
+
+These problems can't be fixed at once by one motivated individual (neither you the developer nor they the user), so more advocacy and [teaching](https://neuropsychology.github.io/psycho.R//2018/08/31/cite_packages.html) is needed. Besides, in the meantime, how do we [adapt software citation guidelines](https://twitter.com/EikoFried/status/1197556604075466753) to realistically accommodate for all situations?
+
+## How rOpenSci tracks package usage
+
+At rOpenSci we monitor the scientific literature to discover uses cases of our packages, that you can browse on our [citations page](/citations).
```
Comment (sckott) [inline content/technotes/2021-03-02-package-citation/index.md:109]: "use cases"

Comment (sckott) [inline content/technotes/2021-03-02-package-citation/index.md:109]: remove comma after packages?

File: content/technotes/2021-03-02-package-citation/index.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+As sometimes packages are used but not listed in the references section, we report any usage of the packages in papers.
+Here's our, well Scott Chamberlain's, workflow.
+
+* Lots of Google Scholar alerts (one per package name, unless that package name will lead to too many false positives) arrive to one gmail address of Scott's.
+* Scott uses [gmailr](https://gmailr.r-lib.org/) and [googlesheets4](https://googlesheets4.tidyverse.org/) to push those up to Google sheets.
+* He then manually clicks through to each link and look for the given package mention and records if a package is actually used (mentioned in the Methods section, and/or cited).
```
Comment (sckott) [inline content/technotes/2021-03-02-package-citation/index.md:115]: You could mention that the citations end up on our citations page https://ropensci.org/citations/ ?

**Review by sckott — COMMENTED** [review]
nice work!

question: Might we add a sidenote about citing data? That is, while you're thinking about making it easier for users to cite your package, if the package connects to data that should be cited, then make it easier to cite the data as well, yes? Maybe that deserves a separate post though?

**Comment by maelle** [comment]
> question: Might we add a sidenote about citing data? That is, while you're thinking about making it easier for users to cite your package, if the package connects to data that should be cited, then make it easier to cite the data as well, yes? Maybe that deserves a separate post though?

Thanks @sckott, I added a sentence about this with a link to https://discuss.ropensci.org/t/citation-of-original-article-when-implementing-specific-methods/2312

File: content/technotes/2021-03-02-package-citation/index.md (hunk 2 above)
Comment (maelle) [reply content/technotes/2021-03-02-package-citation/index.md]: rephrased, thanks.

**Comment by maelle** [comment]
@adamhsparks your feedback would be most welcome as we're using your forum input. :slightly_smiling_face:

**Comment by karthik** [comment]
The post is mostly good. I found the roundabout sentences confusing so I made some edits and pushed them to this PR. Other comments:

> We shall also give some insights on obstacles the users of your package might face, as well as a brief presentation of our own literature surveillance.

Not sure I really understand this sentence. What do you mean by "obstacles the users of your package might face". Also literature surveillance...do you mean literature survey?


> Something we at rOpenSci wouldn't recommend is an on load message in interactive sessions, but it's good to know that [some maintainers resort to this strategy](https://twitter.com/d_olivaw/status/1261439521134182406).

I find this hard to parse and it also sends mixed messages. If we don't recommend it, why are we saying it's good to know that this maintainer does this? I'd say we can rewrite this as:

Although some authors use on load messages to encourage citations, we discourage this practice and recommend that developers highlight this information in their README and documentation.

> Even once you have followed the tips above, well-meaning users of your package might stumble upon

This doesn't sound right. Stumble upon doesn't make sense in this context. I rephrased this. The two tweets for this are also unnecessary (the post is also quite tweet heavy).

**Comment by maelle** [comment]
> The post is mostly good. I found the roundabout sentences confusing so I made some edits and pushed them to this PR. Other comments:

Thanks a ton for the edits @karthik!

> 
> > We shall also give some insights on obstacles the users of your package might face, as well as a brief presentation of our own literature surveillance.
> 
> Not sure I really understand this sentence. What do you mean by "obstacles the users of your package might face". Also literature surveillance...do you mean literature survey?
> 

I rephrased as "In this post, we shall share tips on how to make your R package easy to cite.
We shall also give some insights on hurdles the users of your package might face when wanting to cite your package, as well as a brief presentation of our monitoring of the literature to find use cases of our packages."
Is this clearer?

> > Something we at rOpenSci wouldn't recommend is an on load message in interactive sessions, but it's good to know that [some maintainers resort to this strategy](https://twitter.com/d_olivaw/status/1261439521134182406).
> 
> I find this hard to parse and it also sends mixed messages. If we don't recommend it, why are we saying it's good to know that this maintainer does this? I'd say we can rewrite this as:
> 
> Although some authors use on load messages to encourage citations, we discourage this practice and recommend that developers highlight this information in their README and documentation.
> 

Done, thanks!

> > Even once you have followed the tips above, well-meaning users of your package might stumble upon
> 
> This doesn't sound right. Stumble upon doesn't make sense in this context. I rephrased this. The two tweets for this are also unnecessary (the post is also quite tweet heavy).

Thank you!

File: content/technotes/2021-03-02-package-citation/index.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+  - metadata
+  - packages
+---
+
+Scientists rarely cite research software they use as part of a research project. As a consequence, the software and the time spent developing and maintaining it becomes an invisible scholarly contribution. Furthermore, this lack of visibility means that the incentives to produce high quality, sustainable software is missing. Among many reasons why software is not cited, one is the lack of a clear citation information from package developers. In this tech note we provide some tips on how to make it really easy to cite your software.
+
+
+{{< tweet 1273605411577442304 >}}
+
+In this post, we shall share tips on how to make your R package easy to cite.
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:21]: This sentence is duplicate of previous one. I suggest keeping this and delete the former

File: content/technotes/2021-03-02-package-citation/index.md [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+We shall also give some insights on hurdles the users of your package might face when wanting to cite your package, as well as a brief presentation of our monitoring of the literature to find use cases of our packages.
+We are writing this with the view of complementing our dev guide with the tips... and your feedback, so we are looking forward to your response!
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:23]: link to dev guide? and maybe less colloquial like "our package dev guide"?

Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:23]: restructure this sentence as it's not clear that you're asking for input that will be considered for incorporation into dev guide.  Your conclusion sentence was nice and clear "We encourage you to share your experience and wisdom in the comments below, as we are planning to consolidate them together with our content, to add a new topic to our dev guide."

File: content/technotes/2021-03-02-package-citation/index.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+## Clear citation rules for your lil' tool[^liltool]
+
+<!--html_preserve-->
+{{< figure src ="pexels-cottonbro-4553166.jpg" alt = "Person wearing a cardboard box on their head" link = "https://www.pexels.com/photo/person-in-white-long-sleeve-shirt-holding-brown-cardboard-box-4553166/" caption = "Why didn't you cite my package?! [cottonbro on Pexels](https://www.pexels.com/photo/person-in-white-long-sleeve-shirt-holding-brown-cardboard-box-4553166/)." class = "center" >}}
+<!--/html_preserve-->
+
+To make it really easy for users to cite your package, you should store citation metadata in the expected places and advertise it very clearly.
+
+> "I [advertise the citation info in the README] too, just to make it painfully obvious how to cite the work." [Adam Sparks](/author/adam-sparks/), [rOpenSci forum](https://discuss.ropensci.org/t/should-the-ropensci-dev-guide-include-package-citation-best-practices/1551/18)
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:33]: I'd consider adding Noam's quote first because the [text text] makes it confusing.

> [Personally, I take a “belt-and-suspenders” approach and still put citation things in the README.](https://discuss.ropensci.org/t/should-the-ropensci-dev-guide-include-package-citation-best-practices/1551/17)

File: content/technotes/2021-03-02-package-citation/index.md [hunk 10]
```diff
@@ -0,0 +1,124 @@
...
+
+* Use the CITATION file, as thousands of R packages have done (3,386 out of 16,002 packages on CRAN as of the 15th of January, 2021[^thxmark]). It's easy to create a boilerplate with `usethis::use_citation()`.
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:35]: "use" is slightly unclear cause could mean populate it or use it to cite pkg. Say "Populate" the citation file?

File: content/technotes/2021-03-02-package-citation/index.md [hunk 11]
```diff
@@ -0,0 +1,124 @@
...
+* Archive each release of your GitHub repo on Zenodo and add [Zenodo top-level DOI](https://help.zenodo.org/#versioning) to the CITATION file.
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:36]:
```suggestion
* Archive each release of your GitHub repo on Zenodo and add the [Zenodo top-level DOI](https://help.zenodo.org/#versioning) to the CITATION file.
```

File: content/technotes/2021-03-02-package-citation/index.md [hunk 12]
```diff
@@ -0,0 +1,124 @@
...
+  " nasapower: A NASA POWER Global Meteorology, Surface Solar Energy and Climatology Data Client for R. ",
+  note, ".",
+  " https://CRAN.R-project.org/package=nasapower")
+```
+
+* Direct potential readers to the preferred citation in the README by adding boilerplate text "here's how to cite my package". See e.g. [ecmwfr README](https://github.com/bluegreen-labs/ecmwfr#how-to-cite-this-package-in-your-article).
+
+{{< tweet 1261726909148233728 >}}
+
+Although some authors use on load messages to encourage citations, we discourage this practice and recommend that developers highlight this information in their README and documentation.
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:93]: is "on-load" more clear? This novice had to think about the meaning of "use on load messages"

Maybe it's clear to most of our readers.

File: content/technotes/2021-03-02-package-citation/index.md [hunk 13]
```diff
@@ -0,0 +1,124 @@
...
+
+## Why is it hard to cite software
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:95]:
```suggestion
## Why is it hard to cite software?
```

optional

File: content/technotes/2021-03-02-package-citation/index.md [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+
+Despite your best efforts to encourage users to cite your software, you might still run into challenges. Authors may have limits on the number of references they can cite in a journal or face resistance from their coauthors.
+
+These problems can't be fixed at once by one motivated individual (neither you the developer nor they the user), so more advocacy and teaching is needed. Besides, in the meantime, how do we [adapt software citation guidelines](https://twitter.com/EikoFried/status/1197556604075466753) to realistically accommodate for all situations?
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:99]:
```suggestion
These problems can't be fixed at once by one motivated individual (neither you the developer nor they the user), so more advocacy and teaching is needed. In the meantime, how do we [adapt software citation guidelines](https://twitter.com/EikoFried/status/1197556604075466753) to realistically accommodate for all situations?
```

suggest omitting "Besides" so there's one less thing before the meat of sentence

File: content/technotes/2021-03-02-package-citation/index.md [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+
+## How rOpenSci tracks package usage
+
+At rOpenSci we monitor the scientific literature to discover use cases of our packages that you can browse on our [citations page](/citations).
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:103]: We now use the words "use cases" to describe citations as well as less formal blog posts, gists etc. Could you make it clear when saying "use cases"?

File: content/technotes/2021-03-02-package-citation/index.md [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+As sometimes packages are used but not listed in the references section, we report any usage of the packages in papers.
+Here's Scott Chamberlain's workflow.
+
+* Lots of Google Scholar alerts (one per package name, unless that package name will lead to too many false positives) arrive to one gmail address of Scott's.
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:107]: re-order sentence for active voice so it's more clear, and matches other bullets as a bonus

File: content/technotes/2021-03-02-package-citation/index.md [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+* He then manually clicks through to each link and look for the given package mention and records if a package is actually used (mentioned in the Methods section, and/or cited).
+* He updates [several JSON files of the citations](https://github.com/ropensci-org/ropensci_citations/).
+* [One of these JSON citation files](https://github.com/ropensci-org/ropensci_citations/blob/master/citations_all_parts_clean.json) is used to populate the [citations page](/citations) (via a [Hugo data template in a layout file](https://github.com/ropensci/roweb3/blob/master/themes/ropensci/layouts/citations/list.html)). Any update of the ropensci-org/ropensci_citations repo triggers a Netlify build of our website thanks to a Netlify build hook (basically, an URL) and a [GitHub Actions workflow](https://github.com/ropensci-org/ropensci_citations/blob/master/.github/workflows/rebuild-roweb3.yml) (that simply pings this URL)
+
+As part of our new Moore Foundation funded effort, we are building a system to automatically detect R package citations in literature and append them to package records in R-universe. Stay tuned on our blog for more details.
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:113]:
```suggestion
As part of our [new Moore Foundation funded effort](/blog/2019/11/06/scientific-package-ecosystem/), we are building a system to automatically detect R package citations in literature and append them to package records in R-universe. Stay tuned on our blog for more details.
```

Audience does not know what "R-universe" is. Add appropriate link? Maybe link to Jeroen's rstudio talk as that should be avail before this post

File: content/technotes/2021-03-02-package-citation/index.md [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+
+## Conclusion
+
+In this post we shared guidance on how to help your R package land in the references section of the papers that used it.
+We also mentioned some hurdles even well-meaning users might face, and explained how we track for usage of our packages in the scientific literature.
+We encourage you to share your experience and wisdom in the comments below, as we are planning to consolidate them together with our content, to add a new topic to our dev guide.
+
+Note that this post is not about promoting _usage_ of your package, which is [covered briefly in the dev guide](https://devguide.ropensci.org/marketing.html).
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:121]: put this sentence ("Note that this post...") elsewhere. The previous sentence is good to end the post

File: content/technotes/2021-03-02-package-citation/index.md [hunk 19]
```diff
@@ -0,0 +1,124 @@
...
+
+[^f1000]: Katz DS, Chue Hong NP, Clark T et al. Recognizing the value of software: a software citation guide [version 2; peer review: 2 approved]. F1000Research 2021, 9:1257 (https://doi.org/10.12688/f1000research.26932.2)
+[^liltool]: This is a slightly snarky title responding to a [tweet](https://twitter.com/sk_emptyset/status/1261729463613693952) (isn't the use of "just" in "just a tool to access data" naive at best?). This lively thread also includes, in another place, an infamous [comparison of the tidyverse to a swivel chair](https://twitter.com/georgemsavva/status/1197122359515373574).
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:124]: I read the thread and wonder if you should ask Koen if he's ok with this being embedded. And what about the person who responded? It's a bit rough sounding

File: content/technotes/2021-03-02-package-citation/index.md [hunk 20]
```diff
@@ -0,0 +1,124 @@
...
+
+## Conclusion
+
+In this post we shared guidance on how to help your R package land in the references section of the papers that used it.
+We also mentioned some hurdles even well-meaning users might face, and explained how we track for usage of our packages in the scientific literature.
+We encourage you to share your experience and wisdom in the comments below, as we are planning to consolidate them together with our content, to add a new topic to our dev guide.
+
+Note that this post is not about promoting _usage_ of your package, which is [covered briefly in the dev guide](https://devguide.ropensci.org/marketing.html).
+
+[^f1000]: Katz DS, Chue Hong NP, Clark T et al. Recognizing the value of software: a software citation guide [version 2; peer review: 2 approved]. F1000Research 2021, 9:1257 (https://doi.org/10.12688/f1000research.26932.2)
```
Comment (stefaniebutland) [inline content/technotes/2021-03-02-package-citation/index.md:123]: I can't see where this is cited.

**Review by stefaniebutland — COMMENTED** [review]
Love the post topic, clear intro and content. I think this would be good as a blog post since it's sharing our philosophy and good practice as much as the technical details. This is something for which people look to rOpenSci to set the tone. I propose publishing 2021-02-16 (first available post date).

File: content/technotes/2021-03-02-package-citation/index.md (hunk 12 above)
Comment (maelle) [reply content/technotes/2021-03-02-package-citation/index.md]: I even added a link, good point.

File: content/technotes/2021-03-02-package-citation/index.md (hunk 15 above)
Comment (maelle) [reply content/technotes/2021-03-02-package-citation/index.md]: Made it "use and mentions".

File: content/technotes/2021-03-02-package-citation/index.md (hunk 17 above)
Comment (maelle) [reply content/technotes/2021-03-02-package-citation/index.md]: Added a place-holder link.

File: content/technotes/2021-03-02-package-citation/index.md (hunk 19 above)
Comment (maelle) [reply content/technotes/2021-03-02-package-citation/index.md]: Good point, I removed the embedded tweek and also the snark in the side-note.

File: content/technotes/2021-03-02-package-citation/index.md (hunk 20 above)
Comment (maelle) [reply content/technotes/2021-03-02-package-citation/index.md]: right it is no longer used.

**Comment by maelle** [comment]
@stefaniebutland Many thanks for your review! Remaining items

* Is https://github.com/ropensci/roweb3/pull/116#discussion_r565999516 clearer?
* I'll have to add the link to Jeroen's talk recording.

**Comment by stefaniebutland** [comment]
> Is #116 (comment) clearer?

maybe "uses and mentions"? https://github.com/ropensci/roweb3/pull/116#discussion_r566373663

**Comment by maelle** [comment]
ok, done, so now I'll only have to add the link to Jeroen's talk once it's up.

**Comment by adamhsparks** [comment]
Hi @maelle, this looks great!  Just a few minor suggestions or corrections.

Line 23, "strengthen" is misspelled.

Line 99, I would suggest the following text:

>"Despite your best efforts to encourage users to cite your software, you might still run into challenges. Authors may have limits on the number of references they can cite in a journal or face resistance from their coauthors. Other authors may simply be unaware that they can use `citation("packagename")` to easily retrieve the citation information for an R package or that they even should cite the package because it is not something that has been commonly promoted."

I think that there is a good bit of ignorance out there that the `citation()` function even exists or that people need to cite scientific research tools at all. I've had pushback from co-authors on that alone, that it wasn't deemed necessary (I did it anyway).

**Comment by maelle** [comment]
Thank you @adamhsparks!!

File: content/blog/2021-02-16-package-citation/index.md [hunk 21]
```diff
@@ -0,0 +1,125 @@
...
+  - Scott Chamberlain
+  - Karthik Ram
+description: Some tips to promote scientific citations of your work.
+tags:
+  - citations
+  - metadata
+  - packages
+---
+
+Scientists rarely cite research software they use as part of a research project. As a consequence, the software and the time spent developing and maintaining it becomes an invisible scholarly contribution. Furthermore, this lack of visibility means that the incentives to produce high quality, sustainable software is missing. Among many reasons why software is not cited, one is the lack of a clear citation information from package developers. In this tech note we provide some tips on how to make it really easy to cite your software.
```
Comment (steffilazerte) [inline content/blog/2021-02-16-package-citation/index.md:16]:
```suggestion
Scientists rarely cite research software they use as part of a research project. As a consequence, the software and the time spent developing and maintaining it becomes an invisible scholarly contribution. Furthermore, this lack of visibility means that incentives to produce high quality, sustainable software are missing. Among many reasons why software is not cited, one is the lack of a clear citation information from package developers. In this tech note we provide some tips on how to make it really easy to cite your software.
```

File: content/blog/2021-02-16-package-citation/index.md [hunk 22]
```diff
@@ -0,0 +1,125 @@
...
+
+Although some authors use [on-load messages](https://r-pkgs.org/r.html#when-you-do-need-side-effects) to encourage citations, we discourage this practice and recommend that developers highlight this information in their README and documentation.
+
+## Why is it hard to cite software?
+
+Despite your best efforts to encourage users to cite your software, you might still run into challenges. Authors may have limits on the number of references they can cite in a journal or face resistance from their coauthors. 
+Other authors may simply be unaware that they can use `citation("packagename")` to easily retrieve the citation information for an R package or that they even should cite the package because it is not something that has been commonly promoted.
+
+These problems can't be fixed at once by one motivated individual (neither you the developer nor they the user), so more advocacy and teaching is needed. In the meantime, how do we [adapt software citation guidelines](https://twitter.com/EikoFried/status/1197556604075466753) to realistically accommodate for all situations?
```
Comment (steffilazerte) [inline content/blog/2021-02-16-package-citation/index.md:103]:
```suggestion
These problems can't all be fixed at once by one motivated individual (neither you the developer nor they the user), so more advocacy and teaching is needed. In the meantime, how do we [adapt software citation guidelines](https://twitter.com/EikoFried/status/1197556604075466753) to realistically accommodate for all situations?
```

**Review by steffilazerte — COMMENTED** [review]
Hi @maelle looks good! Just two minor suggestions. And I love reviewing articles: Today I learned about `citHeader()` which would be perfect to include in weathercan to alert people how to cite ECCC data! Thanks :grin:

* [X] post follows Content Guidelines
* [X] post follows Style Guide
* [X] title is in Title Case
* [X] publication date is ok
* [X] alternative text of images is informative
* [X] Twitter metadata looks ok (paste post preview link in [Twitter card validator](https://cards-dev.twitter.com/validator); might have to click twice on "Preview card")
* [X] author metadata is provided with correct folder name
* [X] html not included in pull request of Rmd post
* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md
* [X] YAML subject tags are ok ("tech notes" for tech notes; "community" for non-staff non-editor)

**Comment by steffilazerte** [comment]
Draft tweet, all comments/suggestions/rewrites welcome!

---


[blog] Make Your R Package Easier to Cite

@ma_salmon, @sckottie & @_inundata discuss

- Making it easy to cite your 📦
- Why it's so hard to cite 📦s
- How rOpenSci tracks citations

And invite you to share your insights!

https://ropensci.org/blog/2021/02/16/package-citation/

#rstats #citations

(1/2)

---

🙏 Thank you to @adamhsparks for commenting on an early draft of the blog post

(2/2)

**Comment by maelle** [comment]
Thank you!! And the tweets look good!
